xarray/coding/cftimeindex.py

```diff
@@ -268,29 +268,32 @@ def _partial_date_slice(self, resolution, parsed):
         >>> from cftime import DatetimeNoLeap
         >>> import pandas as pd
         >>> import xarray as xr
-        >>> da = xr.DataArray([1, 2],
-                              coords=[[DatetimeNoLeap(2001, 1, 1),
-                                       DatetimeNoLeap(2001, 2, 1)]],
-                              dims=['time'])
-        >>> da.sel(time='2001-01-01')
+        >>> da = xr.DataArray(
+        ...     [1, 2],
+        ...     coords=[[DatetimeNoLeap(2001, 1, 1), DatetimeNoLeap(2001, 2, 1),]],
+        ...     dims=["time"],
+        ... )
+        >>> da.sel(time="2001-01-01")
         <xarray.DataArray (time: 1)>
         array([1])
         Coordinates:
           * time     (time) object 2001-01-01 00:00:00
-        >>> da = xr.DataArray([1, 2],
-                              coords=[[pd.Timestamp(2001, 1, 1),
-                                       pd.Timestamp(2001, 2, 1)]],
-                              dims=['time'])
-        >>> da.sel(time='2001-01-01')
+        >>> da = xr.DataArray(
```

Comment:
I had thought the standard was a blank line after each result; is that a general standard or was I holding that incorrectly?
(no need to do it this round though)

Reply:
I had the same impression, but the [docs](https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized) say otherwise:

> Expected output cannot contain an all-whitespace line, since such a line is taken to signal the end of expected output. If expected output does contain a blank line, put `<BLANKLINE>` in your doctest example each place a blank line is expected.

I think that means that we only need the newline if we want to continue with normal text.

Reply:
OK. IIUC we _can_ still have blank lines, but it becomes a preference rather than enforced. 
FWIW I do find they make it much easier to read the examples!

Reply:
sounds good. Let's do that in a new PR, though.